README.md

```diff
@@ -121,6 +121,7 @@ The judge can also grade in the languages listed below:
 * GAS x86/x64/ARM
 * Haskell
 * INTERCAL
+* JavaScript (Node.js, V8)
```

Comment:
I suppose for consistency with the C++ variants this should say "(Node.js and V8)".

Reply:
Changed to say `Node.js and V8`!